com.unity.render-pipelines.universal/Editor/ScriptableRendererDataEditor.cs

```diff
@@ -118,7 +118,17 @@ private void DrawRendererFeature(int index, ref SerializedProperty prop)
                     var propertyName = serializedFeature.FindProperty("m_Name");
                     propertyName.stringValue = ValidateName(EditorGUILayout.DelayedTextField(Styles.PassNameField, propertyName.stringValue));
                     if (EditorGUI.EndChangeCheck())
+                    {
                         m_SaveAsset = true;
+
+                        // We need to update sub-asset name
+                        obj.name = propertyName.stringValue;
+                        AssetDatabase.SaveAssets();
+
+                        // Triggers update for sub-asset name change
+                        // TODO: Investigate for better alternative as AssetDatabase for some reason do not trigger this change
```

Comment:
We shouldn't have TODO's in our code if we do not really really really have to.

Reply:
Well this one is tricky, sub asset renaming think could be a bug. However I could not get clarifying response from asset team. Going to try poking them again.

Reply:
I created favro card for this and https://fogbugz.unity3d.com/f/cases/1260165/.